libfastpm/thermalvelocity.c

```diff
@@ -414,12 +413,12 @@ fastpm_split_ncdm(FastPMncdmInitData * nid,
                 dest->v[r][d] = b * dest->v[r][d] + nid->vel[j][d] / (1. + nid->z) / HubbleConstant;
                 
                 dest->x[r][d] = q[d] + b * (x[d] - q[d]);
-            }
+            }*/
             r ++;
         }
     }
 
-    fastpm_store_destroy(sub);
-    fastpm_memory_free(src->mem, mask);
+    //fastpm_store_destroy(sub);
```

Comment:
Since we no longer use this, you can get rid of these commended out lines. (we can always find the older version from git)